src/ghga_service_commons/utils/multinode_storage.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Configuration for multiple object storage nodes"""
+
+from hexkit.providers.s3 import S3Config, S3ObjectStorage
+from pydantic_settings import BaseSettings
+
+
+class ObjectStorageNodeConfig(BaseSettings):
```

Comment:
This is specific to S3:
```suggestion
class S3ObjectStorageNodeConfig(BaseSettings):
```